setup.py

```diff
@@ -49,7 +49,7 @@ def is_requirement(line):
 
 setup(
     name='lti-consumer-xblock',
-    version='2.8.0',
+    version='2.9.0',
```

Comment:
A version bump deserves an entry in the README history :)

Reply:
Sure, will do!